### DISCUSSION TITLE
[Projects] Changes to the labels are not reflected in the issue's history

### DISCUSSION THREAD

**Discussion by guicassolato** [community/community#30817] [Projects and Issues]
When changes are made to the labels of an issue from the popup window that opens to view the details of the issue, on top of a project's board, those changes are not reflected in the issue's history on the issue's page.

This causes an inconsistency for one who reads (audits) the issue's history on the usual issue's page, where changes made to the labels on the issue page show up in the history, but the ones made in the project view do not. Losing the history also leads to an impossibility of matching the history and current state of the issue, e.g.,

1. User **@a** navigates to \<repo>/issues/123 and adds the label "bug" to issue 123.
    - An event "**@a** added 'bug' label X time ago." can be read in the history.
2. User **@b** navigates to a project board that includes issues of \<repo>, clicks to see the details of issue 123 on the board, and removes the label "bug" from issue 123.
3. User **@a** navigates again to \<repo>/issues/123 and sees no label 'bug' on issue 123; the user cannot tell who or when the label was removed from the issue.

As a user, I'd expect all changes made to an issue to be consistently reflected in the issue's history on the issue's page, regardless of the origin of those changes - issue's page itself, a project board's view, or any other way to modify the issue (GitHub API, etc).

**Comment by Samuel-Webster** [comment]
Also doesn't fire the webhook event either I don't think.

**Comment by gm3dmo** [comment]
Engineering team are currently investigating this behaviour.

**Comment by freetimecoder3** [comment]
Same here.  Need to view the history from Project also.
